package/MDAnalysis/topology/TXYZParser.py

```diff
@@ -117,9 +120,16 @@ def parse(self, **kwargs):
         # Guessing time
         masses = guessers.guess_masses(names)
 
+        for n in names:
+             if n.capitalize() in SYMB2Z:
+                 validated_elements.append(n.capitalize())
+             else:
+                validated_elements.append('')
```

Comment:
Do we want to raise a warning about this case? I _think_ we already do this for a few other places.